packages/tasit-action/src/testHelpers/helpers.js

```diff
@@ -56,16 +57,96 @@ export const confirmBalances = async (token, addresses, balances) => {
 // - Try to intercept and parse ethers.BigNumber to BN.js (and vice-versa)?
 // or
 // - Exposes ethers.utils.BigNumber / bigNumberify()?
-export const toBN = ethersBN => {
+const toBN = ethersBN => {
   return new BN(ethersBN.toString());
 };
 
-export default {
+const etherFaucet = async (
+  provider,
+  fromWallet,
+  beneficiaryAddress,
+  amountInWei
+) => {
+  const connectedFromWallet = fromWallet.connect(provider);
+  const tx = await connectedFromWallet.sendTransaction({
+    // ethers.utils.parseEther("1.0")
+    value: "0x0de0b6b3a7640000",
+    to: beneficiaryAddress,
+  });
+  await provider.waitForTransaction(tx.hash);
+};
+
+const erc20Faucet = async (
+  tokenContract,
+  ownerWallet,
+  toAddress,
+  amountInWei
+) => {
+  tokenContract.setWallet(ownerWallet);
+  const mintAction = tokenContract.mint(toAddress, `${amountInWei}`);
+  await mintAction.waitForNonceToUpdate();
+  await confirmBalances(tokenContract, [toAddress], [amountInWei]);
+};
+
+const erc721Faucet = async (tokenContract, ownerWallet, toAddress, tokenId) => {
+  tokenContract.setWallet(ownerWallet);
+  const mintAction = tokenContract.mint(toAddress, tokenId);
+  await mintAction.waitForNonceToUpdate();
+  await confirmBalances(tokenContract, [toAddress], [1]);
+};
+
+const addressesAreEqual = (address1, address2) => {
+  return address1.toUpperCase() === address2.toUpperCase();
+};
+
+const {
+  utils: ethersUtils,
+  constants: ethersConstants,
+  Contract: ethersContract,
```

Comment:
Why capitalize this one unlike the others?

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/237